vhdbuilder/packer/install-dependencies.sh

```diff
@@ -201,7 +201,7 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
 done
 
 AZURE_NPM_VERSIONS="
-1.1.1
```

Comment:
is 1.1.1 not in use?

Reply:
ok 1.1.1 was never in use. 
Can you at least put some description to the release? what is the fix?

Reply:
We didn't pull the latest AKS-E NPM tests so we missed a bug.